pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
```

Comment:
Our scope in tldr pages is limited. So we do not mention the installation step. You can remove this.